src/command/cmd_defs.c

```diff
@@ -2717,6 +2717,21 @@ static const struct cmd_t command_defs[] = {
               "/strophe verbosity 3",
               "/strophe sm no-resend")
     },
+
+    { CMD_PREAMBLE("/privacy",
+                   parse_args, 2, 3, NULL)
+      CMD_MAINFUNC(cmd_privacy)
+      CMD_TAGS(
+              CMD_TAG_CHAT)
+      CMD_SYN(
+              "/privacy logging on|redact|off")
+      CMD_DESC(
+              "Configure privacy settings. ")
```

Comment:
One space too much at the end.
And can you rephrase it to "Configure privacy related settings"?

Reply:
> One space too much at the end. And can you rephrase it to "Configure privacy related settings"?

"Privacy Settings" is correct term, I am not sure why should it be renamed.